### ISSUE TITLE
eth_sendTransaction is dropping transactions

### ISSUE THREAD

**Issue by Coinotron** [ethcore/parity#2547]
I'm running 1.3.5 with these options: --cache-size-db 8192 --tx-gas-limit 500000 --gas-floor-target 1000000 --gasprice 50000000000 --gas-cap 1500000
Mining works fin, but when I try to sent say 10 transactions in a row, few at the beginning are sent ok. Then parity replies with that message:
"There are too many transactions in the queue. Your transaction was dropped due to limit. Try increasing the fee."
 I increased gasprice to 200000000000, queue size to 64000 and tried different queuing strategies with no luck: There are always dropped transactions in pack of 10 transactions sent.

**Comment by trapp** [comment]
I experience the same issue when using `eth_sendRawTransaction`.

**Comment by gituser** [comment]
+1, sometimes transactions are not going through.

here is some snip from gitter:
```
gituser
@gituser
15:14
hey guys, i seems to be getting the same issue with eth_sendTransaction, quite often transaction is not included in the block
but, what I've noticed: if i send exactly same amount to the same address, from the same address I'm getting the the same txhash
and minutes after it's in the block
so it's the correct way to re-broadcast transaction, is that right?
```

**Comment by Atrides** [comment]
I confirm this bug, it looks like parity add own transaction to common queue and limit works here.

**Comment by almindor** [comment]
+1 from me as well, happens on both chains it seems.

**Comment by trapp** [comment]
This is not fixed with 1.3.6. Still getting "There are too many transactions in the queue. Your transaction was dropped due to limit. Try increasing the fee." for local transactions.

**Comment by arkpar** [comment]
@trapp could you run with `-l own_tx=trace` and post the full log?

**Comment by trapp** [comment]
@arkpar 1.3.6 doesn't have the --tx-queue-gas parameter #2553 introduced, so it seems the fix just didn't make it into 1.3.6, is that correct?

**Comment by cenotaph** [comment]
Not sure if I'm having the exact same problem but my contract transactions are not making it to the blockchain - the contract executed a few hundred fine over the past few weeks but suddenly they hang in the pending transactions queue for awhile and then disappear without a trace.

I ran with `-l own_tx=trace`  as asked and got:
```
2016-10-12 21:23:47 UTC  TRACE own_tx  Importing transaction: SignedTransaction { unsigned: Transaction { nonce: 416, gas_price: 32210531854, gas: 1400000, action: Call(87b127ee022abcf9881b9bad6bb6aac25229dff0), value: 120000, data: [121, 198, 80, 104, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 51, 246, 50, 39, 95, 126, 219, 234, 131, 237, 12, 64, 124, 179, 58, 233, 106, 33, 57, 57, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 15] }, v: 28, r: 71331656346093752811338075290754375900394455312007998579101547442562192631391, s: 41131416740459287446670372492783804124344843587121166214442428064684509225007, hash: Cell { value: Some(eef727721edd640d42b90a8a1e60a09e368964c82791793a21e0382441333206) }, sender: Cell { value: None } }
2016-10-12 21:23:47 UTC  TRACE own_tx  Imported transaction to Current (hash: eef727721edd640d42b90a8a1e60a09e368964c82791793a21e0382441333206)
2016-10-12 21:23:47 UTC  TRACE own_tx  Status: TransactionQueueStatus { pending: 10, future: 0 }
```

Also:
```
> eth.getTransaction('0xeef727721edd640d42b90a8a1e60a09e368964c82791793a21e0382441333206')
null
```

This is the just-released parity 1.3.7. I was having the same problem with 1.3.6 but I would at least get a hash returned on getTransaction for a little while after submitting the transaction, with a null blockHash and blockNumber - then a few minutes later the same getTransaction call would just return nill. 

Please let me know if there's any other information I can provide or maybe something I am doing wrong. If it's just a matter of not enough gas (I'm seiding

**Comment by cenotaph** [comment]
Also, if this helps - I just submitted a new transaction, this time running with` --geth`,  and once again:
```
 > eth.getTransaction('0xc458b3f83b1ba424a8b888a4ef83ab41d8ea413bc00a324f37414a081e4c0df9')
null
```

However......

```
> eth.getBlock('pending').transactions[0]
"0xc458b3f83b1ba424a8b888a4ef83ab41d8ea413bc00a324f37414a081e4c0df9"
> eth.getBlock('pending', true).transactions[0]
{
  blockHash: "0x3a590098c74285096eed7fb0af5c5a1dfdb25d77a5f00f3de8ebc9ad260e1fc8",
  blockNumber: 2428796,
  creates: null,
  from: "0x04e1b2ff4ece25f07c67b97757f1c490924e0a04",
  gas: 1400000,
  gasPrice: 31500910288,
  hash: "0xc458b3f83b1ba424a8b888a4ef83ab41d8ea413bc00a324f37414a081e4c0df9",
  input: "0x79c65068000000000000000000000000a570453ce86d5229ae07d8966e2d49990ead9de9000000000000000000000000000000000000000000000000000000000000000f",
  nonce: 416,
  raw: "0xf8af8201a085075599bed083155cc09487b127ee022abcf9881b9bad6bb6aac25229dff08301d4c0b84479c65068000000000000000000000000a570453ce86d5229ae07d8966e2d49990ead9de9000000000000000000000000000000000000000000000000000000000000000f1ca09f098d4e57c8cf88d792cba910c550cff57212d0f1ca9d03e665f45002754398a03de46858c66340fecc48886b4fc141e9203b832f99632042d7873a543edf9a27",
  to: "0x87b127ee022abcf9881b9bad6bb6aac25229dff0",
  transactionIndex: 0,
  value: 120000
}

```

If this is the wrong issue for this I apologise, I can happily start a new one.

**Comment by etherx-dev** [comment]
My transactions never appear on the blockchain either, using latest version of parity.

resyncing now to see if that would help.

**Comment by tomusdrw** [comment]
I believe it should be fixed in 1.3.8 by #2634 and #2616 please reopen if the problem persists on newest version.
